client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -187,6 +187,28 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 		}
 	}
 
+	/**
+	 * Blank Canvas Button
+	 */
+	.build-your-own-cta-wrapper {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: center;
+		padding: 64px;
+		grid-column: 1 / span 3;
+
+		.build-your-own-cta_title {
```

Comment:
I thought the styles don't load correctly. It should be
```diff
-build-your-own-cta_title {
+build-your-own-cta__title {
```

 I only saw the screenshot below

<img width="179" alt="image" src="https://user-images.githubusercontent.com/13596067/186572253-825d87ce-f1ca-4a41-8ce4-dbeef77785b1.png">